.github/workflows/ci.yaml

```diff
@@ -94,6 +94,30 @@ jobs:
           version: v1.55.2
           working-directory: ${{ matrix.directory }}
 
+  govulncheck:
+    name: govulncheck
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Setup go
+      - uses: actions/setup-go@v4
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      - name: govulncheck
+        uses: golang/govulncheck-action@v1
+
+  gosec:
```

Comment:
gosec on a local run reports 5 issues, I would assume enabling the linter without correcting those 5 issues would cause failures in all jobs.